### ISSUE TITLE
Unsure about how to use IExceptionMiddleware

### ISSUE THREAD

**Issue by stephenfournier** [dsuryd/dotNetify#68]
Hi,

I'm not sure how IExceptionMiddleware was intended to be used. When I throw an exception in the get/set of a view model property, the OnException method in the middleware does not get hit. Here is what I have:

```
private double pie1Val1;
public double Pie1Val1
  {
   get { throw new Exception("test1"); return pie1Val1; }
   set { throw new Exception("test2"); pie1Val1 = value; Changed(nameof(Pie1Val1)); }
  }
```

```
	public class MyExceptionMiddleware : IExceptionMiddleware
	{
		public Task<Exception> OnException(HubCallerContext context, Exception exception)
		{
			return Task.FromResult(exception);
		}
	}
```
`config.UseMiddleware<MyExceptionMiddleware>();`

Could you please provide an example on how this middleware is intended to be used?

Thanks!

**Comment by dsuryd** [comment]
This is a bug; thank's for reporting it.  It appears to be broken in the latest release;  try downgrading to v2.2.2.

**Comment by stephenfournier** [comment]
Thanks, is it possible you could push the fix to NuGet?

**Comment by dsuryd** [comment]
I would like to add some more unit tests before I push it.  It'll probably be after the holiday.

**Comment by dsuryd** [comment]
This fix is included in v2.3.1-pre.
